src/main/scala/edu/ie3/util/scala/quantities/SquantsUtils.scala

```diff
@@ -32,9 +32,18 @@ object SquantsUtils {
     ): ElectricPotential = Volts(
       electricPotential.toVolts * that.toEach
     )
-
   }
 
+  implicit class RichSpecificHeatCapacity(
+      specificHeatCapacity: SpecificHeatCapacity
+  ) {
+    def calcEnergyToHeat(
+        volume: Volume,
+        temperatureDelta: Temperature,
+    ): Energy = KilowattHours(
+      specificHeatCapacity.toKilowattHoursPerKelvinCubicMeters * volume.toCubicMeters * temperatureDelta.toKelvinDegrees
+    )
+  }
```

Comment:
There is already a similar method in `SpecificHeatCapacity`. Maybe you could just use that method instead of introducing a new one.